provider/test-programs/cluster/Pulumi.yaml

```
@@ -0,0 +1,31 @@
+name: cluster
+runtime: yaml
+description: A minimal Google Cloud Pulumi YAML program
+resources:
+  serviceAccount:
+    type: gcp:serviceaccount:Account
+    properties:
+      accountId: service-account-1234
+      displayName: Service Account
+  primary:
+    type: gcp:container:Cluster
+    properties:
+      location: us-central1
+      # We can't create a cluster with no node pool defined, but we want to only use
+      #   # separately managed node pools. So we create the smallest possible default
+      #   # node pool and immediately delete it.
```

Looks like something went wrong with formatting here.
```suggestion
      # separately managed node pools. So we create the smallest possible default
      # node pool and immediately delete it.
```